test/presubmit-config-check.sh

```diff
@@ -18,6 +18,10 @@ source $(dirname $0)/../scripts/library.sh
 
 set -e
 
+trap 'echo "--- FAIL: Directly changing production Prow config files is not allowed, please revert the change."' ERR
```

Comment:
let's use add_trap instead

Reply:
Also, the error message should be more actionable, like move the changes to staging dir

Reply:
We don't have `add_trap` function in `library.sh`. Do you want me to add it there?

And I changed the error message.

Reply:
I think it's fine for this script, since these 2 traps don't collide with each other